core/include/detray/propagator/rk_stepper.ipp

```diff
@@ -13,6 +13,7 @@
 
 template <typename magnetic_field_t, typename track_t, typename constraint_t,
           typename policy_t, template <typename, std::size_t> class array_t>
+DETRAY_HOST_DEVICE
```

Comment:
Do you need to add qualifiers for `*.ipp` files?

Reply:
Yes, you do. `__host__` and `__device__` qualifiers need to be put both on the declaration and the implementation of functions. This is what we do in all other projects as well.

Reply:
OK!